src/main/java/usi/si/seart/model/MetricLanguage.java

```diff
@@ -0,0 +1,48 @@
+package usi.si.seart.model;
+
+import lombok.*;
```

Comment:
Do not use wildcard imports, import everything explicitly. This should be configurable somewhere in the IDE settings